README.md

```diff
@@ -256,6 +256,72 @@ to return forces.
 torch_force.setOutputsForces(True)
 ```
 
+Computing energy derivatives with respect to global parameters
+--------------------------------------------------------------
+
+Its possible to query `TorchForce` for the derivative of the energy with respect to global parameters. In order to do so the global parameters must be registered as energy derivatives. This is done by calling `addEnergyParameterDerivative()` for each parameter.
```

Comment:
For the first sentence, how about, "TorchForce can compute derivatives of the energy with respect to global parameters."  Since you don't actually query the TorchForce to get them.